packages/cli/src/lib/bundler/config.ts

```diff
@@ -484,13 +518,16 @@ export async function createBackendConfig(
       extensions: ['.ts', '.mjs', '.js', '.json'],
       mainFields: ['main'],
       modules: [paths.rootNodeModules, ...moduleDirs],
-      plugins: [
-        new LinkedPackageResolvePlugin(paths.rootNodeModules, externalPkgs),
-        new ModuleScopePlugin(
-          [paths.targetSrc, paths.targetDev],
-          [paths.targetPackageJson],
-        ),
-      ],
+      // FIXME: see also https://github.com/web-infra-dev/rspack/issues/3408
+      ...(!useRspack && {
+        plugins: [
+          new LinkedPackageResolvePlugin(paths.rootNodeModules, externalPkgs),
+          new ModuleScopePlugin(
+            [paths.targetSrc, paths.targetDev],
+            [paths.targetPackageJson],
+          ),
+        ],
+      }),
     },
```

Comment:
Could we skip this change for the backend setup? It's deprecated and will be removed soon anyway